src/attributes.md

```diff
@@ -50,6 +50,9 @@ Attributes may be applied to many things in the language:
 * [Generic lifetime or type parameter][generics] accept outer attributes.
 * Expressions accept outer attributes in limited situations, see [Expression
   Attributes] for details.
+* [Function][function], [closure][closure] and [function pointer][function pointer]
```

Comment:
```suggestion
* [Function][functions], [closure] and [function pointer]
```

There is no singular `function` reference.  Also, there's no need to specify the link reference when it is the same as the link label.